static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,121 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
+files may be updated manually as they evolve. Other projects can then share
+these artifacts by downloading (`dvc get`) or importing (`dvc import`) them for
+use in different data processes – and these don't even have to be _DVC
+projects_, as `dvc get` works anywhere in your system.
+
+The advantages of using a data registry are:
+
+- Tracked data is stored in a **centralized** remote location, with the ability
```

Comment:
this list is still quite complicated. So, this is how I see the benefits.

1. Versioning  - you can always download or use in a different project a needed version of the dataset.
2. Persistence - since we use a remote, DVC-controlled bucket, data is secure there. Less chances someone can delete or rewrite a model.
3. Discoverability - Git repo with directories + Github interface give more visibility into datasets
4. Code-like model and datasets lifecycle management - PRs, reviews to update the data, model redeployment when a new one merged in master (we don't need to redeploy the code or change env settings if `dvc.api` is used)
5. Security and audit - you can see who made the changes to the storage, potentially the system can be set (we can think and describe??) so that there is not direct write access to the remote storage, only read one.
6. Reusability - along with import/get this can serve as simple way to reproduce, or up to organizing feature stores.

we can discuss these things and come up with a better list ^^

Reply:
Ok these are all benefits of DVC and I can add them (Versioning is already included, and some of the other with different wording), but what are the special benefits of DVC data registries? Really I can only think of centralization as the special benefit here.

> 3. Discoverability - Git repo with directories...

I don't think we'll really offer data discoverability until `dvc list` is implemented.

> 5. Security and audit - you can see who made the changes to the storage ... not direct write access to the remote storage, only read one.

This is part of what I was describing in the code you asked about in https://github.com/iterative/dvc.org/pull/679#pullrequestreview-303043210... (Removed now)

Reply:
p.s. Don't you think that each of these may be a use case by itself? (Instead of listing all the DVC benefits inside this one use case). Versioning (**Data and Model Files Versioning**) and Reusability (**Share Data and Model Files**) being already there (the latter needing some re-wording).

Reply:
The point of this article is to provide an overview and benefits a _data registry_. We are selling all the benefits of those data registries as, for example, compared to using a plain S3 bucket to store data. I don't think it matters where do those benefits come from. Also this article should be self-contained. It would be too much to expect people to have a list of DVC benefits in their head while reading this.

It's also fine for use case articles to overlap, even to overlap a lot.

Reply:
OK, I understand. I'm just trying to discuss the items as you suggested, so we come up with a better list. **Versioning** is already included, **Discoverability** I don't see; Other ones I just updated in 008e3581 to sound more like your list. (**Reusability** I just kinda copy-pasted, as I'm not sure I get it.)

I was also asking if you agree that Centalization is the one special benefit of data registries vs. regular DVC projects. I left it on top of the list for now.

Please check.